src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -2098,18 +2098,9 @@ def aks_agentpool_add(cmd,      # pylint: disable=unused-argument,too-many-local
                            "use 'aks nodepool list' to get current list of node pool".format(nodepool_name))
 
     upgradeSettings = AgentPoolUpgradeSettings()
-    taints_array = []
+    taints_array = [taint.strip() for taint in node_taints.split(',')]
```

Comment:
`node_taints` is a list. Does a list has `split` method?
```
AttributeError: 'list' object has no attribute 'split'
```

Reply:
Whoops yeah default value should be ""